content/post/GSOC_AMA_report.md

```diff
@@ -0,0 +1,111 @@
+</blockquote>
+<script async src="https://platform.twitter.com/widgets.js" charset="utf-8"></script>
+<footer align=center><cite>A bit of correction, my excited self miscalculated in 2017, we had 2 GSoCer's</cite></footer>
+
+
+It being an honored and proud moment for ALiAS, that this year a **total of 9 students got selected in Google Summer of Code, from Amity University, Noida and all of them, are the members of ALiAS!**
+
+Well, if you don't know what is GSoC. Here's a brief overview.
+
+## What is GSoC?
+
+**GSoC** is a **12-week program**, sponsored by Google  to promote open-source among various college students around the world, where the people selected (Only **1200+ students** total) are assigned to projects from different open-source organizations working over the summer contributing to open-source software. You can learn more about GSoC from [here](https://mixstersite.wordpress.com/2018/04/25/123gsoc/).
+
+<iframe width="560" height="315" align=center src="https://www.youtube.com/embed/QVnN34YGz8s" frameborder="0" allow="accelerometer; autoplay; encrypted-media; gyroscope; picture-in-picture" allowfullscreen></iframe>
+
+**Selected students** get a stipend, loads of experience and skills working under professional mentors. **ALiAS AMA program** has been all about mentoring our members and whosoever that is interested in learning about open-source and willing to contribute to it.
+
+We invited **GSoC mentors**, **admins** and **students** from every major organisation in our networks to get their insights, experience and most importantly answer questions. Our ideology is that **no question is dumb**. And when you believe in this, that's when you know that one could be truly open in this environment.
+
+Henceforth, we had **15 mentors**, for **15 days** straight. For a streak of webinars, live sessions, daily meetups, proposal reviews, chats, doubt clearing, and motivational AMA's.
+
+**AMA (Ask Me Anything) sessions** from the very beginning, have been completely **open** with people coming from together to help, mentor and participate in the program resulting in a nine times the selection this year compared to last 3 years! **This was only possible because of our beliefs, teamwork and true hard work.**
+
+> Be the change you want to see in the world or at least try...
+
+## Activities Related to GSoC done by ALiAS:
+
+###  **Mentoring Sessions**
+
+- Mentoring students throughout the year through various channels like WhatsApp Members Groups, Workshops, Open Meetups, Casual Meetups, and more.
+- Special Sessions by:
```

Comment:
```suggestion
- Special Sessions by:

```
Before:

![Screenshot from 2019-06-02 22-15-38](https://user-images.githubusercontent.com/30147327/58764400-1d951e80-8584-11e9-8042-c8ff66c51403.png)

After:

![Screenshot from 2019-06-02 22-15-59](https://user-images.githubusercontent.com/30147327/58764399-1cfc8800-8584-11e9-93f9-2c9a44565992.png)

They both look kind of odd to me tbh

Reply:
Yes, Hugo is not able to render lists, I think we need to focus on that. What I was going to try next was straight up HTML. Will do

Reply:
Made a change to list system let's see if this works.

Reply:
Working according to docs given here - https://daringfireball.net/projects/markdown/syntax#list

Reply:
Fixed, kinda.